.yarnrc

```diff
@@ -0,0 +1 @@
+registry "https://npm.int.remitly.com/"
```

Comment:
Enable `yarn` tasks, not sure how necessary this is

Reply:
This makes it so you don't have to set the config globally or add it manually each time you install.